react-native/training/src/app/screens/ListBooks/components/Item/index.js

```diff
@@ -0,0 +1,18 @@
+import React, { Component } from 'react';
+
+import { itemsBookList } from '../../../../../propTypes/propTypes';
+
+import Layout from './layout';
+
+class Item extends Component {
```

Comment:
I think that you could change this for functional component and change the name for `ItemContainer`